library/src/main/scala/giter8/GitRepository.scala

```diff
@@ -47,7 +47,7 @@ object GitRepository {
     val NativeUrl: Regex = "^(git[@|://].*)$".r
     val HttpsUrl: Regex  = "^(https://.*)$".r
     val HttpUrl: Regex   = "^(http://.*)$".r
-    val SshUrl: Regex    = "^(ssh://.*)$".r
+    val SshUrl: Regex    = "^ssh://(.*)$".r
```

Comment:
exactly 👍  I don't look up how exactly jgit does, but it works:
![imagen](https://user-images.githubusercontent.com/783959/27904913-937302cc-623e-11e7-8468-00f5d41d5268.png)